pkg/core/nexthash.go

```diff
@@ -14,9 +14,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
```

Comment:
nit: just noticed that this file is called "nexthash" but the type is called "NextPin"